resources/js/screens/queues/index.vue

```diff
@@ -26,17 +31,16 @@
             /**
              * Load the monitored queues.
              */
-            loadQueues() {
+            loadQueues(starting = 0) {
                 this.ready = false;
 
-                this.$http.get(FairQueue.basePath + '/api/monitoring')
+                this.$http.get(FairQueue.basePath + '/api/monitoring?starting_at=' + starting + '&limit=' + this.perPage)
```

Comment:
do we have validation on backend side?

Reply:
No we don't for now

Reply:
let's cast them to int then when using on backend.